src/shared_memory_manager.cc

```diff
@@ -266,14 +266,38 @@ OpenCudaIPCRegion(
   return nullptr;
 }
 
+TRITONSERVER_Error*
+GetCudaDriverEntryPoint(const char* name, void** func_ptr)
+{
+  cudaError_t err = cudaGetDriverEntryPoint(name, func_ptr, cudaEnableDefault);
+  if (err != cudaSuccess) {
+    LOG_ERROR << "Failed to get CUDA driver entry point for " << name << ": "
+              << cudaGetErrorString(err);
+    return TRITONSERVER_ErrorNew(
+        TRITONSERVER_ERROR_INTERNAL,
+        std::string("Failed to get CUDA driver entry point").c_str());
+  }
+  return nullptr;
+}
+
 TRITONSERVER_Error*
 GetCudaSharedMemoryRegionSize(CUdeviceptr data_ptr, size_t& shm_region_size)
 {
+  void* cu_mem_get_address_range = nullptr;
+  void* cu_get_error_string = nullptr;
+  RETURN_IF_ERR(GetCudaDriverEntryPoint(
```

Comment:
Can you add a comment here that why we are making this call? 
Basically, why linking against the cuda driver directly is not an option for Triton.

Reply:
Added the comment.